pyopensprinkler/__init__.py

```diff
@@ -112,6 +113,12 @@ def refresh(self):
             if i not in self._stations:
                 self._stations[i] = Station(self, i)
 
+        fpr0 = self._get_option("fpr0")
+        fpr1 = self._get_option("fpr1")
+        flwrt = self._get_variable("flwrt")
+        flcrt = self._get_variable("flcrt")
+        self._flow_rate = (flcrt * ((fpr1 << 8) + fpr0) / 100) / (flwrt / 60)
```

Comment:
I think all this should be in the property definition itself

Reply:
I did have that originally but it was just inefficient to calculate flow rate every time the property was called as it can only change was a result of `controller.refresh()`. Happy to move it to the property if preferred ?

Reply:
I guess it can be seen the other way as well, you don't need to calculate this unless you ask for it. But I think `refresh()` should just set the state. The efficiency is negligible here I think

Reply:
Done